index.html

```diff
@@ -0,0 +1,329 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio-mobile</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;200;300&family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,400;1,500;1,600;1,700&family=Roboto+Serif:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="head">
+      <h3><a href="index.html">My Logo</a></h3>
+        <img id="union" src="images/Union.png" />
+      <nav>
+        <ul class="links">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+          <li>
+            <a href="#"><img id="mail" src="images/MAIL2.svg" /></a>
+          </li>
+        </ul>
+        </nav>
+    </header>
+    <main>
+      <section class="home">
+        <div class="headline">
+          <h1>Hey There. I'm Cosmas</h1>
+          <h2>I'm a software developer</h2>
+          <p>
+            I can help you build a proffessional website, feature. Look through
+            some of my work and experience and if you are pleased with my work,
+            contact me.
+          </p>
+        </div>
```

Comment:
- [ ] Congratulations on incorporating the feedback from the previous reviewers! However, it seems **that a bug has been introduced in the process**. To ensure that your portfolio page **looks professional and closer** to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=Px1oksowPk0aJMvW-0), I suggest fixing this issue. 😉
After reviewing the problem, **I think the issue might be related to the fixed `margin-top`** you added to your headline. **One possible solution could be to use** a percentage value `(such as 10%)` instead of a fixed pixel value `(10px)`. Additionally, you could remove the `margin-top` property from your `h2` tag. 🙋‍♂️

  _I recommend watching this video to gain a better understanding of the problem and potential solutions._ 🎬

  [5_6_2023_2-29-39 PM.webm](https://user-images.githubusercontent.com/107952724/236643240-84457b5c-5071-4642-b5a7-7f042c7fe5a0.webm)